### PR TITLE
Feature/pl 6066

### PR DESCRIPTION
**Pull request by e136581**

### CODE REVIEW

File: pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/fraser/omregning/ufoeretrygd/Ufoeretrygd.kt [hunk 1]
```diff
@@ -149,19 +149,39 @@ object Ufoeretrygd {
         override fun OutlineOnlyScope<LangBokmalNynorsk, Unit>.template() {
             paragraph {
                 text(
-                    Bokmal to "Du er tidligere innvilget rettighet som ung ufør i uføretrygden din. Denne rettigheten gir deg høyere utbetaling fra og med den måneden du fyller 20 år.",
-                    Nynorsk to "Du har tidlegare fått innvilga rett som ung ufør i uføretrygda di. Denne retten gir deg høgare utbetaling frå og med den månaden du fyller 20 år.",
+                    Bokmal to "Du har tidligere fått innvilget uføretrygd som ung ufør. Fra og med den måneden du fyller 20 år har du rett til høyre minsteytelse.",
+                    Nynorsk to "Du har tidlegare innvilga rett som ung ufør i uføretrygda di. Frå og med månaden du fyller 20 år har du rett til høgare minsteyting.",
                 )
             }
+            paragraph {
```
Comment (hconhegg) [inline pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/fraser/omregning/ufoeretrygd/Ufoeretrygd.kt:156]: Does this text belong here in this phrase? if so, can you include the old reference to this text in the comment above ?     TBU3008, TBU3009, TBU3010, ???

File: pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/vedlegg/OrienteringOmRettigheterAlder.kt [hunk 2]
```diff
@@ -138,43 +140,56 @@ val orienteringOmRettigheterOgPlikterAlder =
...
             barnetilleggVedvirk_innvilgetBarnetillegFellesbarn
-                    and barnetilleggVedvirk_innvilgetBarnetilleggSaerkullsbarn
-                    and not(ektefelletilleggVedvirk_innvilgetEktefelletillegg)
+                and barnetilleggVedvirk_innvilgetBarnetilleggSaerkullsbarn
+                and not(ektefelletilleggVedvirk_innvilgetEktefelletillegg)
         ) {
-            includePhrase(VedleggPlikterRettTilBarnetilleggAP_001)
+            includePhrase(
+                VedleggPlikterRettTilBarnetilleggAP_001(
+                    harTilleggForFlereBarn = harTilleggForFlereBarn
```
Comment (hconhegg) [inline pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/vedlegg/OrienteringOmRettigheterAlder.kt:148]: No need for the first part of harTilleggForFlereBarn = harTilleggForFlereBarn. You can also put this includephrase on one line.

File: pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/vedlegg/OrienteringOmRettigheterAlder.kt [hunk 3]
```diff
@@ -138,43 +140,56 @@ val orienteringOmRettigheterOgPlikterAlder =
...
         }
         showIf(
             ektefelletilleggVedvirk_innvilgetEktefelletillegg
-                    and not(barnetilleggVedvirk_innvilgetBarnetillegFellesbarn)
-                    and not(barnetilleggVedvirk_innvilgetBarnetilleggSaerkullsbarn)
+                and not(barnetilleggVedvirk_innvilgetBarnetillegFellesbarn)
+                and not(barnetilleggVedvirk_innvilgetBarnetilleggSaerkullsbarn)
         ) {
-            includePhrase(VedleggPlikterRettTilEktefelletilleggAP_001(bruker_sivilstand))
+            includePhrase(
```
Comment (hconhegg) [inline pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/vedlegg/OrienteringOmRettigheterAlder.kt:157]: Put all the includephrase on one line

File: pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/vedlegg/OrienteringOmRettigheterAlder.kt [hunk 4]
```diff
@@ -138,43 +140,56 @@ val orienteringOmRettigheterOgPlikterAlder =
...
             barnetilleggVedvirk_innvilgetBarnetillegFellesbarn
-                    or barnetilleggVedvirk_innvilgetBarnetilleggSaerkullsbarn //TODO hva henger denne or egentlig sammen med?
-                    and ektefelletilleggVedvirk_innvilgetEktefelletillegg
+                or barnetilleggVedvirk_innvilgetBarnetilleggSaerkullsbarn //TODO hva henger denne or egentlig sammen med?
+                and ektefelletilleggVedvirk_innvilgetEktefelletillegg
         ) {
-            includePhrase(VedleggPlikterRettTilEktefelletilleggOgBarnetilleggAP_001(bruker_sivilstand))
+            includePhrase(
+                VedleggPlikterRettTilEktefelletilleggOgBarnetilleggAP_001(
+                    bruker_sivilstand,
```
Comment (hconhegg) [inline pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/vedlegg/OrienteringOmRettigheterAlder.kt:170]: Put all the includephrase on one line

**Review by hconhegg — CHANGES_REQUESTED** [review]
Some refactoring is needed.

File: pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/vedlegg/OrienteringOmRettigheterAlder.kt (hunk 4 above)
Comment (e136581) [reply pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/vedlegg/OrienteringOmRettigheterAlder.kt]: Corrected and pushed

**Review by hconhegg — APPROVED** [review]

**Pull request merged**